spec/features/user_deletes_account_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+feature 'user deletes account', %(
```

Comment:
I would suggest adding another scenario to this feature that tests the behavior if I am logged in as user A and try to delete the account for user B (perhaps by navigating manually to the account settings page for user B). Do you restrict this?